packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.types.ts

```diff
@@ -274,4 +277,10 @@ export interface IGroupDividerProps {
 
   /** Determines if the group selection check box is shown for collapsed groups. */
   isCollapsedGroupSelectVisible?: boolean;
+
+  /** Allows the user to provide custom class name for the header chevron. */
+  headerChevronClassName?: string;
```

Comment:
what is the classname used for, just styling? we will be moving this component to css-in-js soon enough and will have a properly typed style interface for the chevron.

Reply:
Yes, it's just for styling. I'll remove it since there is a plan to add this customization.